packages/kbn-management/settings/components/form/storybook/get_form_story.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+
+import { FieldDefinition, SettingType } from '@kbn/management-settings-types';
+import { getFieldDefinitions } from '@kbn/management-settings-field-definition';
+import { Form } from '../form';
+import { getSettingsMock, uiSettingsClientMock } from '../mocks';
+
+export interface StoryProps {
+  /** True if saving settings is enabled, false otherwise. */
+  isSavingEnabled: boolean;
+  /** True if settings require page reload, false otherwise. */
+  requirePageReload: boolean;
+}
+
+/**
+ * Utility function for returning a {@link Form} Storybook story.
+ * @returns A Storybook Story.
+ */
+export const getFormStory = () => {
```

Comment:
nit: when you only have one story-- rather than a number of stories with different subjects-- you can just put the story inline, (e.g. `form.stories.tsx`)

Reply:
That makes sense, thanks! Added with [d67ccda](https://github.com/elastic/kibana/pull/166460/commits/d67ccda9627025a7cc95c93cd43428d554c74b54)